README.md

```diff
@@ -0,0 +1,220 @@
+         which is also randomized.
+    >
+    >   **Note**: Once you have cast your ballot you **WILL NOT**
+        be able to modify it. Please consider the nominations carefully
+        and cast your ballot once you have come to a decision.
+
+- "Email text" - Include basic information on the vote, where to find more
+  information, and a notice regarding the end of voting.
+
+    >   The 2023 election for the Board of Directors
+        of the Python Software Foundation is open.
+        The full list of candidates is available at
+        https://python.org/nominations/elections/2023-python-software-foundation-board/nominees.
+    >
+    >   Voting will close Friday, June 30, 2023 at 11:59 pm UTC as previously announced.
+        See https://www.python.org/nominations/elections/2023-python-software-foundation-board/nominees/
+        for details on the election as well as helpful countdown clocks.
+    >
+    >   **Note**: Once you have cast your ballot you **WILL NOT**
+        be able to modify it. Please consider the nominations carefully
+        and cast your ballot once you have come to a decision.
+
+- "Language" - `English`
+
+- "Expert mode" - You MUST enable expert mode! Which will reveal all options
+  following this bullet.
+
+- "Show results during voting" - "no"
+
+- "Voting start date" - Optionally this to the date that voting will start.
+  This will automatically send ballots at 12:01 AM that day
+  (In the timezone specified in your account profile).
+
+- "Voting stop date" - Optionally this to the date that voting will end.
+  This will automatically close the election at 11:59 PM that day
+  (In the timezone specified in your account profile).
+
+- "Automatic reminders" - "yes"
+
+- "Anonymous voting" - "yes"
+
+- "Candidate names" - Add the names of candidates as they appear on the
+  nomination statements, one-per-line.
+
+- "Method" - "Approval Voting"
+
+- "Number of winners" - Per [Bylaws Section 5.5][2], 3 or 4 depending on
+  the outgoing cohort size for a given year. It *may* be more if any
+  additional board vacancies have been created.
+
+- "Shuffle candidate order": "yes"
+
+- There *may* be additional questions such as bylaws changes, if so
+  use the `(+) Add Another Contest` button and use judgement to determine
+  how to add them.
+
+### Setting up Voters
+
+The Election Adminitsrator (EA) will receive a file with the first name, last
+name, membership type, and email address of a voter.
+In order to add voters to an election,
+OpaVote requires they be uploaded in an ASCII text file in the format::
+
+    email-address
+
+For example::
+
+    graffatcolmingov@gmail.com
+    ewdurbin@pyfound.org
+
+The format of the file the election administrator typically gets is of the
+format::
+
+    FirstName,LastName,MembershipType,Email
+
+There is a tool in this repository to handle it called
+``convert-exported-csv-for-opavote.py``. This will generate the necessary
```

Comment:
Should this script be in this PR or is that a follow up?

Reply:
It'll be added once I have the first report!